Makefile

```diff
@@ -80,8 +80,21 @@ releases: release/access-slack release/access-jira release/access-mattermost rel
 .PHONY: build-all
 build-all: access-slack access-jira access-mattermost access-pagerduty access-gitlab access-email terraform event-handler
 
+.PHONY: release-version
+release-version: update-api-version update-version terraform/gen-schema update-tag
```

Comment:
I'm not sure `update-tag` should be a part of this target though? I usually do it like this:
- Call `make update-version VERSION=vX.Y.Z`  to update versions, push to Github, open a PR.
- Once the PR gets merged, update local master and only then call `make update-tag VERSION=vX.Y.Z`.

If we bundle them together, it will tag the branch and start building it before the actual PR with the changes gets merged.